### PR TITLE
fix new bridge creator comaptible

### PR DESCRIPTION
**Pull request by Jason-W123**

### CODE REVIEW

File: scripts/transferOwnership.ts [hunk 1]
```diff
@@ -15,32 +11,22 @@ export async function transferOwner(
...
-  const L1AtomicTokenBridgeCreator__factory = new ethers.Contract(
-    TOKEN_BRIDGE_CREATOR,
-    L1AtomicTokenBridgeCreator.abi,
-    l2Deployer
-  )
-  const l1TokenBridgeCreator =
-    L1AtomicTokenBridgeCreator__factory.connect(l2Deployer)
+
+  // Read the JSON configuration
+  const configRaw = fs.readFileSync('outputInfo.json', 'utf-8')
```
Comment (Jason-W123) [inline scripts/transferOwnership.ts:25]: Should we read from `outputInfo.json`?

Comment (gzeoneth) [reply scripts/transferOwnership.ts:25]: I think you can just read the `executorContractAddress` from the token bridge factory

Comment (Jason-W123) [reply scripts/transferOwnership.ts:25]: But we need to get `L2AtomicTokenBridgeFactory` address? and for now we don't store it in any file now

**Review by gzeoneth — APPROVED** [review]
Lgtm

**Pull request merged**